2.0/s2i/bin/run

```diff
@@ -10,6 +10,8 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_vars=$(cgroup-limits.py) ; export $export_vars
```

Comment:
We should probably not have the ".py" suffix in the file name, and simple run it as an executable in the PATH...
There are other utils coming from sti-base that have no extension in the file name.